README.md

```diff
@@ -106,11 +116,11 @@ For example, to start the browser example app with a specific URL, one can run
 It's possible to package the repo's example application with `electron-builder`. After running `yarn` in the repo root, do:
 
 ```bash
-  $ cd electron-app
+  $ cd examples/electron
   $ yarn package
 ```
 
-  The configured Linux packaging(s) will be generated folder `electron-app/dist`
+The configured Linux packaging(s) will be generated folder `examples/electron/dist`
```

Comment:
Here's the other required correction to make the existing packaging code work as expected.

Reply:
I plan to fix it in the other PR as well.